build/Targets/VSL.Versions.targets

```diff
@@ -73,7 +73,7 @@
   </Choose>
 
   <!-- Returns the current build version. Used in .vsixmanifests to substitute our build version into them -->
-  <Target Name="GetBuildVersion" Outputs="$(BuildVersion)" />
+  <Target Name="GetBuildVersion" Outputs="$(RoslynSemanticVersion).$(BuildNumberPart1.Trim())$(BuildNumberPart2.Trim())" />
```

Comment:
Also, what _is_ `BuildNumberPart1` and `BuildNumberPart2`? Why were they separated? Are they lonely, which is why we're putting them back together?